inc/Menu/Item/Revert.php

```diff
@@ -23,4 +23,12 @@ public function __construct() {
         $this->svg = DOKU_INC . 'lib/images/menu/06-revert_replay.svg';
     }
 
+    /** @inheritdoc */
+    public function getLinkAttributes($classprefix = 'menuitem ') {
+        global $lang;
+        $attr = parent::getLinkAttributes($classprefix);
+        $attr['onclick'] = 'javascript:return confirm("'.hsc($lang['js']['restore_confirm']).'")';
```

Comment:
1. I am afraid `javascript:` prefix is not needed: https://stackoverflow.com/questions/372159/do-you-ever-need-to-specify-javascript-in-an-onclick
2. The proper escape function here should be `addslashes` rather than `hsc` or you will get:

```
Really &quot;&lt;1&gt; &quot;restore this version?
```

Reply:
JavaScript should not be attached inline. Use jQuery to attach the handler.

Reply:
Thanks for the PR review. In last commit have moved the confirm dialog in `` lib/scripts/behaviour.js``